examples/README.md

```diff
@@ -271,6 +263,21 @@ The following platforms are currently supported:
 
 To add a new platform definition, add a new class to `krun/platform.py`.
 
+## Developer Mode
+
+If you are working in Krun itself, you may find the `--develop` switch useful:
+
+  * Krun will not run the system prerequisite checks. Checks relating to CPU
+    governers,  CPU scalers, CPU temperatures, tickless kernel, etc.
```

Comment:
`Krun will not run the system prerequisite checks. 
For example, checks relating to CPU governers,  CPU scalers, CPU temperatures, tickless kernel, etc.`